crates/symbolicator-proguard/src/interface.rs

```diff
@@ -1 +1,62 @@
+use std::sync::Arc;
 
+use serde::{Deserialize, Serialize};
+use symbolic::common::DebugId;
+use symbolicator_service::types::Scope;
+use symbolicator_sources::SourceConfig;
+
+#[derive(Debug, Clone)]
+pub struct SymbolicateJvmStacktraces {
+    pub scope: Scope,
+    pub sources: Arc<[SourceConfig]>,
+    pub exceptions: Vec<JvmException>,
+    pub modules: Vec<JvmModule>,
+    /// Whether to apply source context for the stack frames.
+    pub apply_source_context: bool,
+}
+
+#[derive(Debug, Default, Clone, Deserialize, Serialize, PartialEq, Eq)]
+pub struct JvmFrame {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub function: Option<String>,
+
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub filename: Option<String>,
```

Comment:
Can we add a description to all of these attributes?

Reply:
Yeah, will do